netbout-client/src/test/java/com/netbout/client/NbRule.java

```diff
@@ -45,6 +45,7 @@
     /**
      * Get user.
      * @return User
+     * @checkstyle NonStaticMethodCheck (10 lines)
```

Comment:
@mkordas I am not sure the method is supposed to be static. If yes, no problem to make it static then.